src/main/kotlin/io/emeraldpay/dshackle/config/StreamingConfig.kt

```diff
@@ -0,0 +1,12 @@
+package io.emeraldpay.dshackle.config
+
+const val DEFAULT_CHUNK_SIZE = 1048576 // default is 1 MB
+
+data class StreamingConfig(
```

Comment:
Maybe use application.yaml and env variable to override? I think 99.999% providers should have one default value